apps/estuary/mix.exs

```diff
@@ -34,6 +34,7 @@ defmodule Estuary.MixProject do
       {:divo, "~> 1.1", only: [:dev, :integration]},
       {:divo_kafka, "~> 0.1.5", only: [:dev, :integration]},
       {:distillery, "~> 2.1"},
+      {:placebo, "~> 1.2", only: [:dev, :test, :integration]},
```

Comment:
Shall we put all the dependencies in alphabetical order, it will be easy to find out in future

Reply:
sure